### PR TITLE
Create scroll_of_thunder_v.lua

### PR DESCRIPTION
**Pull request by some1here**
Added Item function

### CODE REVIEW

File: scripts/globals/items/scroll_of_thunder_v.lua [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-----------------------------------------
+--	ID: 4776
+--	Scroll of thunder v
```
Comment (TeoTwawki) [inline scripts/globals/items/scroll_of_thunder_v.lua]: Single space here, not TABKEY

File: scripts/globals/items/scroll_of_thunder_v.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+--	Teaches the black magic thunder v
+-----------------------------------------
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+	return target:canLearnSpell(168);
```
Comment (TeoTwawki) [inline scripts/globals/items/scroll_of_thunder_v.lua]: 4 SPACE here, not TABKEY

File: scripts/globals/items/scroll_of_thunder_v.lua [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+end;
+
+-----------------------------------------
+-- OnItemUse
+-----------------------------------------
+
+function onItemUse(target)
+	target:addSpell(168);
```
Comment (TeoTwawki) [inline scripts/globals/items/scroll_of_thunder_v.lua]: 4 SPACE here, not TABKEY

**Comment by TeoTwawki** [comment]
please make same adjustments in your other pull requested scripts

**Comment by bendangelo** [comment]
@TeoTwawki Stop being like that. It looks fine to me.

**Comment by TeoTwawki** [comment]
@bendangelo being like what? why have a standard if we're not going to ask people to use it?

**Comment by some1here** [comment]
I used the same set Form as https://github.com/DarkstarProject/darkstar/blob/e41c17615cdbcba03023f5961b875fd93e84e1fe/scripts/globals/items/scroll_of_aeroja.lua

**Comment by TeoTwawki** [comment]
I apologize but we've been trying to get people to stop doing that

**Pull request closed without merging**